cmd/checkmarxExecuteScan.go

```diff
@@ -555,17 +555,17 @@ func zipFolder(source string, zipFile io.Writer, patterns []string) error {
 		}
 		defer file.Close()
 		_, err = io.Copy(writer, file)
+		fileCount++
 		return err
 	})
-
+	log.Entry().Infof("Zipped %d files", fileCount)
 	return err
 }
 
 func filterFileGlob(patterns []string, path string, info os.FileInfo) bool {
-	for index := 0; index < len(patterns); index++ {
-		pattern := patterns[index]
+	for _, pattern := range patterns {
```

Comment:
You could also *trim* the `pattern` here.

Reply:
The string operation here would be executed for every file in all directories and for each pattern, therefore it is more economical to do an equivalent string operation only once at the beginning.